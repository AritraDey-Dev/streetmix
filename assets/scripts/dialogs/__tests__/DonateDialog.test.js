/* eslint-env jest */
import React from 'react'
import { renderWithReduxAndIntl } from '../../../../test/helpers/render'
import DonateDialog from '../DonateDialog'

describe('DonateDialog', () => {
  it('renders', () => {
    const wrapper = renderWithReduxAndIntl(
      <DonateDialog closeDialog={jest.fn()} />
    )
    expect(wrapper.asFragment()).toMatchSnapshot()
  })
})
