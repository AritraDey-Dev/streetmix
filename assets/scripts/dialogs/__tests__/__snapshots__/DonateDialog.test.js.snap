// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DonateDialog renders 1`] = `
<DocumentFragment>
  <div
    class="donate-dialog"
  >
    <h1>
      Streetmix needs your help!
    </h1>
    <div
      class="donate-dialog-text"
    >
      <p
        class="donate-dialog-lede"
      >
        Streetmix is a community-supported project. Unlike other “free” services on the Internet, we do not show ads or inject trackers that abuse your privacy. Instead, we depend on your support to keep Streetmix up and running.
      </p>
      <p>
        Please consider supporting us with a one-time or recurring contribution. Your donation will pay for our servers and cloud infrastructure, supporting other non-profits, advocacy groups, and students working to improve our streets. Additional revenue will be invested into further development of new features. Our expenses are completely transparent, so you can see exactly how your contributions are used by the Streetmix team.
      </p>
    </div>
    <p>
      <a
        class="button-like donate-dialog-button"
        href="https://opencollective.com/streetmix"
        rel="noopener noreferrer"
        target="_blank"
      >
        Donate to Streetmix
      </a>
    </p>
    <p>
      <a
        href="#"
      >
        No thanks
      </a>
    </p>
  </div>
</DocumentFragment>
`;
